exams/api_test.py

```diff
@@ -130,6 +130,37 @@ def test_exam_authorization(self):
             course=self.course_run.course
         ).exists() is True
 
+    def test_exam_authorization_attempts_consumed(self):
+        """
+        test exam_authorization when user passed and paid, but used all their attempts
+        """
+        create_order(self.user, self.course_run)
+        mmtrack = get_mmtrack(self.user, self.program)
+        self.assertTrue(mmtrack.has_paid(self.course_run.edx_course_key))
+        self.assertTrue(mmtrack.has_passed_course(self.course_run.edx_course_key))
+        old_run = ExamRunFactory.create(course=self.course_run.course)
+        ExamAuthorizationFactory.create_batch(
+            2,
```

Comment:
Maybe replace hard-coded 2 in tests with 'ATTEMPTS_PER_PAID_RUN', just in case it changes in the future?